app/views/overview/_virtual-machine-row.html

```diff
@@ -47,9 +47,9 @@ <h3>
   </div>
   <div class="list-pf-expansion collapse" ng-if="row.expanded" ng-class="{ in: row.expanded }">
     <div class="list-pf-container">
-      <div class="expanded-section resource-details">
-        <h3>Details</h3>
-        <div class="row">
+      <div class="row-expanded-top overview-ovm">
```

Comment:
I think these classes should stay the same so the pattern is consistent.  Why is the change necessary?